src/frontend/komponenter/Felleskomponenter/Venstremeny/sider.ts

```diff
@@ -160,3 +152,24 @@ export const erViPåUlovligSteg = (pathname: string, behandlingSide?: ISide) =>
 
     return false;
 };
+
+export const finnSteg = (behandling: IBehandling): BehandlingSteg => {
+    const erHenlagt = inneholderSteg(behandling, BehandlingSteg.HENLEGG_SØKNAD);
+
+    if (erHenlagt) {
+        if (inneholderSteg(behandling, BehandlingSteg.SEND_TIL_BESLUTTER))
+            return BehandlingSteg.SEND_TIL_BESLUTTER;
+        if (inneholderSteg(behandling, BehandlingSteg.VILKÅRSVURDERING))
+            return BehandlingSteg.VILKÅRSVURDERING;
+        return BehandlingSteg.REGISTRERE_SØKNAD;
```

Comment:
For at dette ble litt mer dynamisk kunne man vel hentet det siste/første ikke_utførte steget?

Reply:
Denne var litt komplisert å få riktig, i dette tilfelle er det ikke IKKE_UTFØRT steg man skal få men steget før, Og det skal være en av disse tre (ellers virker ikke videre funksjonalitet)

Reply:
hmm, hvorfor steget før? Hvis man henlegger på vilkårsvurdering skal vel vilkårsvurdering være IKKE_UTFØRT etter henleggelsen?